Makefile

```diff
@@ -586,6 +586,13 @@ test-interactive-search: $(search_tests:=.search)
 test-interactive-help:
 	$(KEVM) help
 
+test-kevm-vm:
+	bash -c " \
+	  kevm-vm 8888 127.0.0.1 & \
+	  netcat localhost 8888 -q 2 < tests/hello_proto.bin; \
```

Comment:
let's move this to `tests/vm/hello_proto.bin` instead, to keep the test directory a bit cleaner